src/components/CatImageUploadForm.tsx

```diff
@@ -77,23 +78,40 @@ const CatImageUploadForm: React.FC<Props> = ({ uploadCatImage }) => {
     }
   };
 
+  const createDisplayErrorMessage = (error: Error) => {
+    const errorName = error.name;
+
+    // TODO errorNameを型安全に取り出せるようにリファクタリングする
+    switch (errorName) {
+      case 'UploadCatImageSizeTooLargeError':
+        return '画像サイズが大きすぎます。お手数ですが2MB以下の画像を利用して下さい。';
+      case 'UploadCatImageValidationError':
+        return '画像フォーマットが不正です。お手数ですが別の画像を利用して下さい。';
+      default:
+        return 'アプロード中に予期せぬエラーが発生しました。お手数ですが、しばらく時間が経ってからお試し下さい。';
```

Comment:
@keitakn ここは`アップロード`かな？🐱
```suggestion
        return 'アップロード中に予期せぬエラーが発生しました。お手数ですが、しばらく時間が経ってからお試し下さい。';
```

Reply:
@kobayashi-m42 typoしてた:sweat_drops:ありがとう🐱